src/core/core.js

```diff
@@ -323,9 +323,6 @@ export default class Core {
         window.performance.mark('yext.answers.verticalQueryResponseRendered');
       }).catch(error => {
         console.error('Vertical search failed with the following error: ' + error);
```

Comment:
Since we catch more than API failures, could we update this copy? If a card fails to render, the search didn't fail. Maybe something like "The following problem was encountered during Vertical search: "

Reply:
sure thing